### ISSUE TITLE
[bug]

### ISSUE THREAD

**Issue by pifeifei** [khanamiryan/php-qrcode-detector-decoder#138]
php: 7.4.32

PHP Parse error:  syntax error, unexpected '|', expecting variable (T_VARIABLE) in https://github.com/khanamiryan/php-qrcode-detector-decoder/blob/1.0.6/lib/QrReader.php#L16

```php

final class QrReader
{
	private readonly \Zxing\BinaryBitmap $bitmap;
	private readonly \Zxing\Qrcode\QRCodeReader $reader;
	private \Zxing\Result|bool|null $result = null;  // php 7.4 不支持这样的变量声明
}
```

**Comment by GenieTim** [comment]
Yes, 1.0.6 was removed from GitHub for this reason (the PHP 8.1 version was not required in composer.json), but the maintainer did not manage to remove it from composer. Just downgrade to 1.0.5.2.

P.S.: PHP 7.4 will not have any support anymore in ca. 3 days. I suggest you upgrade asap.
